apps/class-solid/src/components/Experiment.tsx

```diff
@@ -89,6 +89,38 @@ function RunningIndicator() {
   );
 }
 
+function DownloadExperiment(props: { experiment: Experiment }) {
```

Comment:
Awesome! 

I always thought of the download as including only the config, but I agree it makes sense to also have the experiment name etc included. Not sure about the output though, we need to think about that. Also, I wonder if it makes sense to include a version/PID of the classconfig jsonschema at some point.

I guess my main concern is that you can reuse the downloaded config also outside the web app, e.g. when running class in nodejs.

Reply:
Opened #14